fea-rs/test-data/parse-tests/bad/glyphsapp_number_bad.ERR

```diff
@@ -0,0 +1,23 @@
+error: expected '$ident' or '${predicate}'
+in ./test-data/parse-tests/bad/glyphsapp_number_bad.fea at 3:13
+  | 
+3 |     pos A B $0;
+  |              ^
+
+error: expected ident or number
+in ./test-data/parse-tests/bad/glyphsapp_number_bad.fea at 4:22
+  | 
+4 |     pos C D ${pad * * 2};
+  |                       ^
```

Comment:
Is `* * ` exponentiation? If not, shouldn't we get something like expected value for the second * per the `value [op value]*` rule?

Reply:
no, we don't support exponentiation; this is a test that we error when there are two operators in a row.